packages/infra/lib/lambdas-nested-stack.ts

```diff
@@ -346,6 +361,15 @@ export class LambdasNestedStack extends NestedStack {
         alarmAction: props.alarmAction,
         lahieSftpIngestionBucket,
       });
+
+      this.hl7AlohrSftpIngestionLambda = this.setupAlohrSftpIngestionLambda({
+        lambdaLayers: props.lambdaLayers,
+        vpc: props.vpc,
+        secrets: props.secrets,
+        config: props.config,
+        alarmAction: props.alarmAction,
+        alohrSftpIngestionBucket,
+      });
```

Comment:
see existing patterns - i believe you should be able to just check if ownProps.config.hl7Notification !== undefined && ownProps.config.hl7Notification?.AlohrSftpIngestionLambda !== undefined or something and get proper type inference so you don't need to throw a bunch of random ass errors in `setupAlohrSftpIngestionLambda` to unwrap all the optional values.